openjdk-8.yaml

```diff
@@ -0,0 +1,222 @@
+package:
+  name: openjdk-8
```

Comment:
should we call out the icedtea provenance? @kaniini

Reply:
We do in the description.